weston-ivi-shell/src/ivi-controller.c

```diff
@@ -1809,6 +1810,18 @@ layer_event_remove(struct wl_listener *listener, void *data)
     }
 }
 
+static bool
+is_desktop_surface_create(struct ivishell *shell,
```

Comment:
Function name does not convey its functionality. This function check for a valid surface id, please name it accordingly.